source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureEks.cs

```diff
@@ -305,6 +306,32 @@ public void AuthorisingWithAmazonAccount(bool runAsScript)
                 ExecuteCommandAndVerifyResult(TestableKubernetesDeploymentCommand.Name);
             }
         }
+        
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        [WindowsTest] // We are having an issue with this test running on Linux. The test successfully executes on Windows.
```

Comment:
I was having massive issues getting these tests to pass on Debian on TeamCity. The tests pass fine correct, but the AWS CLI on Debian is having problems.

We also realized the other CLI tests in `KubernetesContextScriptWrapperFixture.cs` had the `[WindowsTest] attribute applied to them, with a comment saying they have issues on Linux.

After discussion with @eddymoulton, we decided that the code is working (as proved by the windows test), but to get unblocked, just not run this test on Linux

Reply:
> the code is working (as proved by the windows test)

Confidence was also increased by this same test working locally on my ubuntu based machine.